src/exe/colmap.cc

```diff
@@ -1466,8 +1467,9 @@ int RunVocabTreeRetriever(int argc, char** argv) {
   std::string vocab_tree_path;
   std::string database_image_list_path;
   std::string query_image_list_path;
-  int num_images = -1;
-  int num_verifications = 0;
+  std::string output_index_path;
+  retrieval::VisualIndex<>::QueryOptions query_options;
+  bool spatial_verification = false;
```

Comment:
It might be cleaner to add this to the ``QueryOptions``, but I can do that at some point later as well.